WoWsShipBuilder.Web/Components/TopMenu.razor

```diff
@@ -66,6 +66,8 @@
         <MudNavMenu Bordered Color="Color.Info">
             <MudNavLink Icon="@shipBuilderIcon" Match="NavLinkMatch.All" Href="/">Homepage</MudNavLink>
             <MudNavLink Icon="@Icons.Filled.SsidChart" Href="/charts">Dispersion and Ballistic Charts</MudNavLink>
+            <MudNavLink Icon="@Icons.Filled.SsidChart" Href="/shipComparison">Ship Comparison</MudNavLink>
```

Comment:
Fix link and also ensure that ship comparison links in this file are working everywhere. Ensure correct spacing between all navigation items